java-security-test/README.md

```diff
@@ -1,18 +1,22 @@
 # SAP CP Java Security Test Library
 
 ## Description
```

Comment:
In description we should also mentioned that you can use it with Spring Boot too, otherwise if I as a Spring Boot app developer read this and don't look at the very bottom of spring section I assume that this is only For Java EE apps